app/forms/support_interface/block_submission_form.rb

```diff
@@ -11,7 +11,11 @@ def save(fraud_match_id)
 
       return false unless valid?
 
-      FraudMatch.find(fraud_match_id).update!(blocked: true, fraudulent: true)
+      FraudMatch.find(fraud_match_id).candidates.each do |candidate|
+        return unless candidate.update(submission_blocked: true)
```

Comment:
This will return false if the `Candidate` is invalid — do we want to bail under those circumstances? (Should we assume it's going to work and get an exception if something went wrong?)

Reply:
The controller handles the boolean return value although getting a `false` return value from `update` here seems unlikely.